ansible/roles/elasticsearch/tasks/rolling_restart.yml

```diff
@@ -18,6 +18,8 @@
   sudo: true
   service: name=elasticsearch state=started
 
+- debug: msg="Sometimes we try to start the node too soon. If hung start node manually"
```

Comment:
the output is 

```
TASK [debug] *******************************************************************
ok: [192.168.33.16] => {
    "msg": "Sometimes we try to start the node too soon. If hung start node manually"
}
```

seemed like the easiest way to add a message, but i didn't look super hard either